app/components/settings/text_setting.tsx

```diff
@@ -55,7 +55,7 @@ type Props = {
     maxLength?: number;
     optional: boolean;
     onChange: (value: string) => void;
-    value: string;
+    value: string | undefined;
```

Comment:
nit: any reason why it's not 
```suggestion
    value?: string;
```
to match the ones above?

Reply:
First one I did, I forgot to update :sweat_smile: